test/integration/graql/reasoner/query/OntologicalQueryIT.java

```diff
@@ -1,330 +0,0 @@
-        Session session = server.sessionWithNewKeyspace();
-        loadFromFileAndCommit(resourcePath, "matchingTypesTest.gql", session);
-        try (Transaction tx = session.transaction(Transaction.Type.WRITE)) {
-            String basePattern = "$x isa service;" +
-                    "$y isa service;" +
-                    "(owner: $x, capability: $xx) isa has-capability; $xx isa $type;" +
-                    "(owner: $y, capability: $yy) isa has-capability; $yy isa $type;" +
-                    "$y != $x;";
-
-            String simpleQuery = "match " +
-                    basePattern +
-                    "get $x, $y;";
-            String queryWithExclusions = "match " +
-                    basePattern +
-                    "$meta type entity; $type != $meta;" +
-                    "$meta2 type thing; $type != $meta2;" +
-                    "$meta3 type capability-type; $type != $meta3;" +
-                    "get $x, $y, $type;";
-
-            List<ConceptMap> simpleAnswers = tx.execute(Graql.parse(simpleQuery).asGet(), false);
-            List<ConceptMap> simpleAnswersInferred = tx.execute(Graql.parse(simpleQuery).asGet());
-            List<ConceptMap> answersWithExclusions = tx.execute(Graql.parse(queryWithExclusions).asGet(), false);
-            List<ConceptMap> answersWithExclusionsInferred = tx.execute(Graql.parse(queryWithExclusions).asGet());
-            assertFalse(simpleAnswers.isEmpty());
-            assertFalse(answersWithExclusions.isEmpty());
-            assertCollectionsNonTriviallyEqual(simpleAnswers, simpleAnswersInferred);
-            assertCollectionsNonTriviallyEqual(answersWithExclusions, answersWithExclusionsInferred);
-        }
-    }
-
-    @Test
-    public void instancesOfSubsetOfTypesExcludingGivenType() {
-        try(Transaction tx = session.transaction(Transaction.Type.READ)){
-            String queryString = "match $x isa $type; $type sub entity; $type2 type noRoleEntity; $type2 != $type; get $x, $type;";
-
-            List<ConceptMap> answers = tx.execute(Graql.parse(queryString).asGet(), false);
-            List<ConceptMap> answersInferred = tx.execute(Graql.parse(queryString).asGet());
-
-            assertFalse(answers.isEmpty());
-            assertCollectionsNonTriviallyEqual(answers, answersInferred);
-        }
-    }
-
-    //TODO need to correctly return THING and RELATION mapping for %type
-    @Ignore
-    @Test
-    public void allInstancesAndTheirType() {
-        try(Transaction tx = session.transaction(Transaction.Type.READ)) {
-            String queryString = "match $x isa $type; get;";
-
-            List<ConceptMap> answers = tx.execute(Graql.parse(queryString).asGet());
-            assertCollectionsNonTriviallyEqual(answers, tx.execute(Graql.parse(queryString).asGet(), false));
-        }
-    }
-
-    @Test
-    @Ignore //TODO: re-enable this test once we figure out why it randomly fails
-    public void allRolePlayerPairsAndTheirRelationType() {
```

Comment:
have we migrated this? Did we try to get it to run a bunch of times? Does it fail?

Reply:
It is migrated to schema-queries.feature, line 123. It consistently fails, every time.